public/resources/core/js/controllers/product-offering.price.controller.js

```diff
@@ -22,19 +22,54 @@
         vm.CHARGE_PERIODS = Offering.TYPES.CHARGE_PERIOD;
         vm.CURRENCY_CODES = Offering.TYPES.CURRENCY_CODE;
         vm.PRICES = Offering.TYPES.PRICE;
+        vm.PRICE_ALTERATIONS = Offering.TYPES.PRICE_ALTERATION;
+        vm.PRICE_CONDITIONS = Offering.TYPES.PRICE_CONDITION;
+
+        vm.priceAlteration = 0;
```

Comment:
It is better for readability having an enumeration for this instead of just numbers